src/test/java/org/opensearch/neuralsearch/processor/TextChunkingProcessorIT.java

```diff
@@ -176,6 +181,43 @@ public void testTextChunkingProcessor_withCascadePipeline_successful() {
         }
     }
 
+    @SneakyThrows
+    public void testTextChunkingProcessor_withIgnoreMissing() {
```

Comment:
Please use unit tests instead of integ tests to check the logic of your classes. 
Integ tests are expensive to setup and run, in the past high number of tests that deploy models lead to all kinds of issues in CI.